iam/cloud-client/snippets/create_service_account.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains code samples that demonstrate how to get create service account.
+
+# [START iam_create_service_account]
+from typing import Optional
+
+from google.cloud import iam_admin_v1
+from google.cloud.iam_admin_v1 import types
+
+
+def create_service_account(project_id: str, account_id: str, display_name: Optional[str] = None) -> None:
+    """
+    Creates a service account.
+    project_id: ID or number of the Google Cloud project you want to use.
+    account_id: ID or email which will be unique identifier of the service account
```

Comment:
Service accounts come with a name of following format: `<username>@<project-name>.iam.gserviceaccount.com`

Is that parameter supposed to be the <username> part only, or the full e-mail format identifier? Or both are accepted?